src/pages/home/base/mobile-core/lifecycle/index.md

```diff
@@ -30,25 +35,17 @@ CocoaPods<br/>(iOS)
 
 <Tabs query="platform=ios-pods&task=add"/>
 
-## Register Lifecycle with Mobile Core
-
-<TabsBlock orientation="horizontal" slots="heading, content" repeat="4"/>
-
-Kotlin<br/>(Android)
-
-<Tabs query="platform=android-kotlin&task=register"/>
+### Initialize Adobe Experience Platform SDK with Lifecycle Extension
 
-Java<br/>(Android)
+Next, initialize the SDK by registering all the solution extensions that have been added as dependencies to your project with Mobile Core. For detailed instructions, refer to the [initialization](/src/pages/home/getting-started/get-the-sdk/#2-add-initialization-code) section of the getting started page.
 
-<Tabs query="platform=android-java&task=register"/>
+Using the `MobileCore.initialize` API to initialize the Adobe Experience Platform Mobile SDK simplifies the process by automatically registering solution extensions and enabling lifecycle tracking.
 
-Swift<br/>(iOS)
+<InitializeSDK query="componentClass=TabsBlock"/>
 
-<Tabs query="platform=ios-swift&task=register"/>
-
-Objective-C<br/>(iOS)
+<InlineAlert variant="info" slots="text"/>
 
-<Tabs query="platform=ios-objc&task=register"/>
+Lifecycle is automatically enabled with the `MobileCore.initialize` API. The following `lifecycleStart` and `lifecyclePause` instructions apply only to [manual extension registration](/src/pages/home/getting-started/get-the-sdk/#b-manual-extension-registration-using-mobilecoreregisterextensions-api).
```

Comment:
Can you reword and move this alert below the next heading? 

Lifecycle tracking is enabled by default when the `MobileCore.initialize API` is used and lifecycle extension is included as an app dependency.  The following instructions only apply when using [manual extension registration](/src/pages/home/getting-started/get-the-sdk/#b-manual-extension-registration-using-mobilecoreregisterextensions-api) to register Lifecycle extension.